src/main/java/com/dougnoel/sentinel/configurations/Configuration.java

```diff
@@ -405,6 +405,19 @@ public static String operatingSystem() {
         return operatingSystem;
     }
     
+    /**
+     * Returns the directory to use for images set in the config file, on the command line, or a default directory in logs.
+     * @return String the directory to use for test image output
+     */
+    public static String imageDirectory() {
+    	String imageDirectory = Configuration.toString("imageDirectory");
+        if(imageDirectory == null) {
+    		imageDirectory = "logs/images";
+    	}
+        
+        return imageDirectory;
+    }
+    
```

Comment:
Move this method to the FileManager. That's the object that needs to know what the path is and handle it.

Reply:
I moved it and updated the javadocs comments. Changed the calls to it to reflect this change.